src/app/CommandSender.h

```diff
@@ -580,8 +588,12 @@ class CommandSender final : public Messaging::ExchangeDelegate
 
     chip::System::PacketBufferTLVWriter mCommandMessageWriter;
 
+#if CHIP_CONFIG_COMMAND_SENDER_BUILTIN_SUPPORT_FOR_BATCHED_COMMANDS
+    PendingResponseTrackerImpl mPendingResponseTracker;
```

Comment:
OK, so `mPendingResponseTracker` and `mpPendingResponseTracker` are way too similar.  It took me a few attempts to figure out why the code in the constructor was sane.

How about calling this `mDefaultPendingResponseTracker` or something?  Or `mNonTestPendingResponseTracker`?